### PR TITLE
Fix (some of the) data sync docs flaws

### PR DESCRIPTION
**Pull request by jhellar**
# Motivation
https://issues.jboss.org/browse/AEROGEAR-7894

### CODE REVIEW

**Comment by psturc** [comment]
Looks good to me so far. Let's wait for all tasks in [release testing epic](https://issues.jboss.org/browse/AEROGEAR-7792) to be completed.

File: modules/ROOT/pages/sync/provisioning.adoc [hunk 1]
```diff
@@ -29,6 +31,7 @@ If the Postgres password was generated, retrieve the password using:
 ----
 oc describe dc postgres-data-sync -n <myproject> | grep POSTGRESQL_PASSWORD
 ----
+Wait for the database provisioning to finish before executing this command.
```
Comment (finp) [inline modules/ROOT/pages/sync/provisioning.adoc]: @pmacko1  didn't we have something like this recently? This warning should be before the command.

Comment (pmacko1) [reply modules/ROOT/pages/sync/provisioning.adoc]: Yep, in metrics, see comment from @aidenkeating 
https://docs.google.com/document/d/1q49WNxWQPnUWF6_97ot305coDoI3sgYJ9QUz89JKXzk/edit
and yours in data sync
https://docs.google.com/document/d/1L5o7L715Q_pO0y3VGY0enQ7dqdmAA53qkqAV7IVLTfg/edit#

File: modules/ROOT/pages/_partials/installing-mobile-services.adoc [hunk 2]
```diff
@@ -17,7 +17,7 @@ NOTE: Due to an link:https://github.com/openshift/origin/issues/18747[OpenShift
 +
 NOTE: Installation on macOS with Docker Machine has not been tested, use the link:https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md#macos-with-docker-for-mac[Docker for Mac] instructions when installing OpenShift.
 +
-** Use link:https://www.openshift.org/download.html[OpenShift client tools, window="_blank"] version 3.9 or later
+** Use link:https://www.openshift.org/download.html[OpenShift client tools, window="_blank"] version 3.9
```
Comment (pmacko1) [inline modules/ROOT/pages/_partials/installing-mobile-services.adoc]: adresses https://github.com/aerogear/mobile-docs/issues/452

File: modules/ROOT/pages/sync/provisioning.adoc [hunk 3]
```diff
@@ -20,6 +20,8 @@ When provisioning a {service-name} Service, you are prompted to set the followin
 |Postgres Storage Size (Gb)|Size of persistent volume for Postgres (default value is recommended)
 |Postgres User|User name that will be used to connect to postgres ('user' will be used if blank)
 |Postgres Password|Password to connect to Postgres (generated if blank)
+|Query depth limit|Limit the complexity of the queries solely by their depth
```
Comment (pmacko1) [inline modules/ROOT/pages/sync/provisioning.adoc]: what is this @jhellar ?

File: modules/ROOT/pages/sync/provisioning.adoc (hunk 1 above)
Comment (pmacko1) [reply modules/ROOT/pages/sync/provisioning.adoc]: https://github.com/aerogear/mobile-docs/issues/459

File: modules/ROOT/pages/sync/resolvers_ui.adoc [hunk 4]
```diff
@@ -65,6 +65,8 @@ For example, a request mapping to retrieve all the comments for a particular `No
 select * from Comments where NoteId = '{{ context.parent.id }}'
 ```
 
+NOTE: For in-memory data sources link:https://github.com/louischatriot/nedb[NeDB] is used. See link:https://github.com/louischatriot/nedb#api[NeDB API] to write request mappings.
```
Comment (pmacko1) [inline modules/ROOT/pages/sync/resolvers_ui.adoc]: improve this sentence

Comment (pmacko1) [reply modules/ROOT/pages/sync/resolvers_ui.adoc]: https://github.com/aerogear/mobile-docs/issues/460

**Pull request merged**